ApplicationPattern.yaml

```diff
@@ -6109,6 +6151,65 @@ paths:
                                 properties:
                                   request:
                                     type: string
+                      - description: 'generic response profile'
+                        type: object
+                        required:
+                          - uuid
+                          - profile-name
+                          - response-profile-1-0:response-profile-pac
+                        properties:
+                          uuid:
+                            type: string
+                          profile-name:
+                            type: string
+                            enum:
+                              - 'action-profile-1-0:PROFILE_NAME_TYPE_ACTION_PROFILE'
+                              - 'file-profile-1-0:PROFILE_NAME_TYPE_FILE_PROFILE'
+                              - 'integer-profile-1-0:PROFILE_NAME_TYPE_INTEGER_PROFILE'
+                              - 'response-profile-1-0:PROFILE_NAME_TYPE_GENERIC_RESPONSE_PROFILE'
+                              - 'string-profile-1-0:PROFILE_NAME_TYPE_STRING_PROFILE'
+                          response-profile-1-0:response-profile-pac:
+                            type: object
+                            required:
+                              - response-profile-capability
+                              - response-profile-configuration
+                            properties:
+                              response-profile-capability:
+                                type: object
+                                required:
+                                  - operation-name
+                                  - field-name
+                                  - datatype
+                                properties:
+                                  operation-name:
+                                    type: string
+                                  field-name:
+                                    type: object
+                                    minProperties: 1
+                                    maxProperties: 1
```

Comment:
Similar to above comment, adding "additionalProperties: false" restriction would help in this situation to restrict entry of incorrect property in request-body

Reply:
Changed as proposed despite it is useless at a GET.